_data/sponsors.yml

```diff
@@ -65,4 +65,29 @@
   img: /assets/img/sponsors/MITLibraries.png
   link: https://libraries.mit.edu/
   level: silver
-  annotation: 'Closed Captioning & Post-Conference Video Production'
\ No newline at end of file
+  annotation: 'Closed Captioning & Post-Conference Video Production'
+
+- name: Disability, Mental Health, and Accessibility in Libraries
+  img: /assets/img/sponsors/DMHALibraries.png
+  link:
```

Comment:
Added a commit with some logic to determine if a sponsor logo be wrapped in a link. If `link:` is null, it disables the common UI feedback that indicates a link (e.g cursor: pointer;)